src/js/fetch.js

```diff
@@ -1,10 +1,10 @@
-import { showLoader } from "./loader";
+import { showLoader } from './loader';
 //fetching function
 export const fetchPopular = async () => {
-	showLoader();
-	const response = await fetch(
-		"https://api.themoviedb.org/3/movie/popular?api_key=964358699754c21d74c014b561cf196c",
-	);
-	const popularMovies = await response.json();
-	return JSON.stringify(popularMovies);
```

Comment:
czy jesteś pewien że tam nie powinno być JSON.stringify ?

Reply:
Już Ci mówię: jak zwracałem to w ten sposób, cały czas wyskakiwał błąd undefined w funkcji displayMovies() bo ona ma w ciele forEacha, który nie chciał czytać tego arraya z popularMovies.results jako array, tylko czytał chyba całość jako string i nie ładował mi niczego. Można to pewnie zwrócić ze stringify, tak jak było, tylko pewnie później już w index.js trzeba by to było sparse'ować, ale jak ruszyło w ten sposób, który obecnie zacommitowałem, to już nie grzebałem dalej, żeby bardziej nie popsuć 🤣